src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.template.Figure;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area:" + getArea()
+                + " sq. units, radius: " + getRadius()

```

Comment:
```suggestion
                + " sq. units, radius: " + radius
```
you have direct access to those fields - dont use getters